### ISSUE TITLE
Collapsible blocks without keyboard focus indicators in HTML, violating SC 2.4.7 in WCAG 2.2

### ISSUE THREAD

**Issue by jimjyang** [asciidoctor/asciidoctor#4656]
When using collapsible blocks (`[%collapsible]`), the resulting collapsible blocks in HTML don't have keyboard focus indicators, thus violating [Success Criterion 2.4.7 Focus Visible in WCAG 2.2](https://www.w3.org/TR/WCAG22/#focus-visible). 

Any workarounds?

**Comment by mojavelinux** [comment]
You are free to customize the stylesheet to your liking. See https://docs.asciidoctor.org/asciidoctor/latest/html-backend/custom-stylesheet/

In the future, please ask questions about usage in the project chat at https://chat.asciidoctor.org. Thanks.
